canonical-web-and-design/maas-ui
# Determine chassis power types from api.

## Done
- Replaced hardcoded list of chassis power types with dynamically filtered list from api.

## QA
- Go to `/r/machines/add` and check that all the power types can be selected from the dropdown.
- Go to `/r/machines/chassis/add` and check that the list of power types is filtered by `chassis: true` according to `general.powerTypes.data` in state.

## Fixes
Fixes #1012
